spec/routing/project_routing_spec.rb

```diff
@@ -30,29 +30,29 @@
 
 describe ProjectsController, type: :routing do
   describe 'index' do
-    it { expect(get('/projects')).to route_to(controller: 'projects', action: 'index') }
-    it { expect(get('/projects.atom')).to route_to(controller: 'projects', action: 'index', format: 'atom') }
-    it { expect(get('/projects.xml')).to route_to(controller: 'projects', action: 'index', format: 'xml') }
+    it { expect(get('/projects')).to        route_to(controller: 'projects', action: 'index') }
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.